recipes/pocketsphinx/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+pulseaudio-libs-devel
```

Comment:
It would be nice to have this as an actual conda-forge package.

Reply:
I asked about one [Gitter](https://gitter.im/conda-forge/conda-forge.github.io?at=5b637fd5945df30dc1462060), and it seemed like the response was that `pulseaudio` would be a pain to package.  I'm sure it's possible, but I'm also sure that it's over my head! 🙂

Reply:
totally fair!